lib/kubernetes-deploy/kubernetes_resource/custom_resource_definition.rb

```diff
@@ -41,8 +43,28 @@ def prunable?
       prunable == "true"
     end
 
+    def timeout_for_children
+      @definition.dig("metadata", "annotations", CHILD_CR_TIMEOUT_ANNOTATION)&.to_i
+    end
+
+    def rollout_params
+      return nil unless rollout_params_string
```

Comment:
So the annotation has to be present, possibly empty for the deploy to look at the status?

Reply:
Yes, the presence of an empty object will default to using the preset values for success/failure. These can be overridden piecemeal as needed. The defaults set now are the base conditions we decided on in cloudbuddies

Reply:
OK, would be good to make that clear in the docs.

Reply:
Can you please provide a sample of the complete annotation for reviewers?